server/listen.go

```diff
@@ -29,6 +29,64 @@ import (
 	"github.com/netflix/rend/protocol"
 )
 
+type tcpListener struct {
+	listener net.Listener
+}
+
+func (l *tcpListener) Accept() (net.Conn, error) {
+	return l.listener.Accept()
+}
+
+func (l *tcpListener) ModifyConnSettings(conn net.Conn) error {
```

Comment:
I am not exactly sure why this is a method on the listener object. But do you think it should be renamed as ```ModifyRemoteConnSettings``` in that case?